.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,63 @@
+name: Update release notes
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on: 
+   workflow_dispatch:
+   schedule:
+     - cron: '00 00 * * *'
+     # At 00:00 everyday
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  update-bom:
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.1.0
+        with:
+          ref: main
+          
+      - name: Check existing PRs
+        run: |
+          if gh pr list --repo https://github.com/yangyansong-adbe/aep-mobile-sdkdocs --json title | grep 'Updating release notes for BOM'; then
```

Comment:
This is currently pointing to your fork.

Reply:
Good catch. This is hard coded for testing. Let me update it now.